go/common/rollups.go

```diff
@@ -22,3 +22,11 @@ func (r *ExtRollup) Hash() L2RootHash {
 	r.hash.Store(v)
 	return v
 }
+
+func (r *ExtRollup) ToExtBatch() *ExtBatch {
```

Comment:
I guess this method existing is just a temporary thing?

Reply:
It has one usage atm - when you get back the genesis rollup, you need to convert it to a batch to distribute. But it may go away in the future, yeah.